src/BucketListItem.jsx

```diff
@@ -21,8 +45,34 @@ export function BucketListItem(props) {
             });
           }}
         />
-        {props.item.text}
+        {/* Item text */}
+        {isEditable() ? (
+          <input
+            type="text"
+            value={editableText()}
+            onChange={(e) => {
+              setEditableText(e.target.value);
+            }}
+          />
+        ) : (
+          props.item.text
+        )}
       </label>
+
+      {/* Buttons */}
+      {isEditable() ? (
+        <div>
+          <button onClick={bucketTextEdit}>Save</button>
+          <button onClick={handleIsEditable}>Cancel</button>
+        </div>
+      ) : (
+        <div>
+          {!props.item.complete && (
+            <button onClick={handleIsEditable}>Edit</button>
+          )}
+          <button>Delete</button>
```

Comment:
Remove delete. This issue is only for Edit. One PR for one issue, no extra code should be added to the PR.

Reply:
Dada, I have removed the delete button. After pulling this repo I styled the edit button with tailwind CSS and saved the edited text to local storage using the saveWish function from the util file.

### Screenshot or example output
![image](https://github.com/tapascript/solid-bucket-list/assets/105661852/e0d42290-55d4-41ce-aeab-591bfac2f369)